src/components/Tooltip/Tooltip.tsx

```diff
@@ -56,7 +60,7 @@ export function Tooltip<
   const [wrapTooltip, setWrapTooltip] = useState(false)
   const [positionStyles, setPositionStyles] = useState({})
 
-  const { position, wrapperclasses, className } = props
+  const { wrapperclasses, className } = props
```

Comment:
One thing that might be nice is to continue to do prop destructing in one place. 

I don't have a strong opinion on whether `position` and its default value is moved here, or whether `wrapperClasses` and `className` destructuring are moved to the functional component argument. Other than perhaps that most of our components destructure in the component arg, so that'd be a point for consistency. 

@aligg What do you think?

Reply:
@brandonlenz I just pushed up a change to move the prop destructuring into the component arg. If that looks good lmk and hopefully will be good to merge!

Reply:
LGTM! Thank you!